charts/operator_helm3/README.md

```diff
@@ -0,0 +1,102 @@
+# Rookout
+
+[Rookout](http://rookout.com/) gets data from your live code, as it runs. Extract any piece of data from your code and pipeline it anywhere, in realtime, even if you’d never thought about it beforehand or created any instrumentation to collect it.
+
+## TL;DR;
+
+```bash
+helm repo add rookout https://helm-charts.rookout.com
+helm repo update
+helm install --name my-release rookout/operator_helm3 --set operator.token=YOUR_ORGANIZATIONAL_TOKEN
```

Comment:
I think --name isnt featured in helm3

Reply:
also you have `rookout/operator_helm3` and `rookout/operator`, in this PR.
I think that sticking to one helm would be easier to maintain

Reply:
I know I suggested the 2 versions but it seems that no one is doing that kind of pattern.
Doing in with2 steps like cert-manager is better